macos/FluentUI/Button/Button.swift

```diff
@@ -424,7 +424,7 @@ class ButtonCell: NSButtonCell {
 			y += CGFloat(yOffsetSign) * (titleSize.height + titleToImageSpacing - titleToImageVerticalSpacingAdjustment) / 2
 		}
 
-		return NSRect(x: x, y: y, width: imageSize.width, height: imageSize.height)
+		return NSRect(x: x.rounded(.towardZero), y: y.rounded(.towardZero), width: ceil(imageSize.width), height: ceil(imageSize.height))
```

Comment:
Assuming this is the intend to use floor and ceil, I would probably do the following. 
```swift
return NSRect(
x: x.rounded(.towardZero), 
y: y.rounded(.towardZero), 
width: imageSize.width.rounded(. toNearestOrAwayFromZero), 
height: imageSize.height.rounded(. toNearestOrAwayFromZero)
```

Reply:
> > your commit history tells me you didn't create a new branch for this PR?
> 
> I did create a new branch, but re-used an old branch name. GitHub claims that branch deletion "can't be undone", but this doesn't seem to be true in the sense that no history of the deleted branch is retained. I wasn't expecting a bunch of irrelevant commit history to show up in this PR.

are you actually deleting your "origin" branch before reusing the same name branch for your pr?

Reply:
either consistently use floor and ceil /or use .rounded for all of them.